BUILD.bazel

```diff
@@ -117,9 +117,9 @@ go_test(
         "read_test.go",
         "tombstones_test.go",
     ],
-    data = [":testdata"],
+    data = ["//testdata"],
     embed = [":zoekt"],
-    embedsrcs = ["//testdata:search_result_1.pb"],  #keep
+    embedsrcs = ["//testdata:testdata"],  #keep
```

Comment:
This was the painful part. Gazelle doesn't seem to be able to figure out how this embed works.